static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -338,7 +348,30 @@ const SPECIAL_FIELDS: SpecialFields = {
             }}
             items={items}
           />
-        </Container>
+        </RightAlignedContainer>
+      );
+    },
+  },
+  minidump: {
+    sortField: null,
+    renderFunc: data => {
+      const attachments: Array<IssueAttachment & {url: string}> = data.attachments;
+
+      const minidump = attachments.find(
+        attachment => attachment.type === 'event.minidump'
```

Comment:
could there be more than one minidump?

Reply:
Should be one minidump per event, that's how the current all events tab works as well